packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -480,6 +481,947 @@ describe('postgresql - missing database', () => {
   })
 })
 
+describe('postgresql views re-introspection warnings', () => {
+  const connectionString = process.env.TEST_POSTGRES_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
+
+  function computeSetupParams(warningCode: EngineArgs.ViewWarningCodes) {
+    const setupParams: SetupParams = {
+      connectionString,
+      // Note: at this location there is a setup.sql file
+      // which will be executed a SQL file so the database is not empty
+      dirname: path.join(
+        __dirname,
+        '..',
+        '__tests__',
+        'fixtures',
+        're-introspection',
+        'postgresql',
+        `views-warning-${warningCode}`,
+      ),
+    }
+    return setupParams
+  }
+
+  describe('postgresql views 21', () => {
+    const warningCode = 21
+    const setupParams = computeSetupParams(warningCode)
+
+    beforeAll(async () => {
+      await tearDownPostgres(setupParams!).catch((e) => {
+        console.error(e)
+      })
+    })
+
+    beforeEach(async () => {
+      await setupPostgres(setupParams!).catch((e) => {
```

Comment:
Are we sure we want to silence the error and proceed if setup fails? Chances that any of the tests will succeed in this case are pretty slim, imho.

Reply:
Hm, I'm less familiar as to what alternatives we have / what would be more idiomatic/helpful here

Reply:
I'd just drop that `.catch`:
- if `setupPostgress` throws exception, we would still see it.
- if we could not setup postgress because of exception, running the rest of the tests is likely pointless - they won't find the database they'd expect.

Reply:
Would you recommend the same thing then for the `afterEach` `tearDown`?